fs/layer/node.go

```diff
@@ -88,7 +88,34 @@ var opaqueXattrs = map[OverlayOpaqueType][]string{
 	OverlayOpaqueUser:    {"user.overlay.opaque"},
 }
 
-func newNode(layerDgst digest.Digest, r reader.Reader, blob remote.Blob, baseInode uint32, opaque OverlayOpaqueType, logFSOperations bool) (fusefs.InodeEmbedder, error) {
+const (
+	// fuse operations
+	fuseOpGetattr     = "Getattr"
```

Comment:
If we can rename these node operation as `node.opName`, then we can reuse them in state file reports instead of using strings. (e.g., `n.fs.s.report(fmt.Errorf("node.Open: %v", err)`).